runtime/compiler/trj9/runtime/JitRuntime.cpp

```diff
@@ -1155,6 +1169,8 @@ JIT_HELPER(_jitProfileLongValueX);
 JIT_HELPER(_jitProfileBigDecimalValueX);
 JIT_HELPER(_jitProfileStringValueX);
 JIT_HELPER(_jitProfileAddressX);
+JIT_HELPER(_jProfileValueX);
+JIT_HELPER(_jProfileLongValueX);
```

Comment:
Would it be more clear if these were `_jProfile32BitValue` and `_jProfileValue64BitValue` instead? The names seem orthogonal at the moment.

Reply:
Thanks for the suggestion, I have updated them accordingly.